core/src/test/java/io/grpc/ClientInterceptorsTest.java

```diff
@@ -351,6 +352,33 @@ protected void checkedStart(ClientCall.Listener<RespT> responseListener, Metadat
     ArgumentCaptor<Status> captor = ArgumentCaptor.forClass(Status.class);
     verify(listener).onClose(captor.capture(), any(Metadata.class));
     assertSame(error, captor.getValue().getCause());
+
+    // Make sure nothing bad happens after the exception.
+    ClientCall<?, ?> noop = ((CheckedForwardingClientCall<?, ?>)interceptedCall).delegate();
```

Comment:
This is for the latest build on master:
https://coveralls.io/builds/3843611/source?filename=core%2Fsrc%2Fmain%2Fjava%2Fio%2Fgrpc%2FClientInterceptors.java

You can look at builds with type "push" at https://coveralls.io/github/grpc/grpc-java for master's current state.